708statMech.tex

```diff
@@ -122,16 +123,16 @@ \subsection*{The central problem of thermodynamics}
 \begin{enumerate}
 \item Extensivity: ($S$ is an extensive variable) If 1 and 2 are thermodynamic systems then $S_{1\cup2}=S_1+S_2$.
 \item Convexity: If $X^1=(X_0^1,X_1^1,\ldots,X_r^1)$ and  $X^2=(X_0^2,X_1^2,\ldots,X_r^2)$ are two thermodynamic states of the same system then for $0\leq\alpha\leq1$
-$$ S((1-\alpha)X^1+\alpha X^2)\geq (1-\alpha)S(X^1)+\alpha S(X^2) $$.
+$$ S((1-\alpha)X^1+\alpha X^2)\geq (1-\alpha)S(X^1)+\alpha S(X^2) $$
 I.e. the entropy of a linear combination of states is greater than or equal to the same linear combination of entropies of the individual states.
 A consequence of this is that if we take derivatives with respect to $\alpha$ at $\alpha=0$ we get
 $$\frac{\partial}{\partial \alpha} S((1-\alpha)X^1+\alpha X^2) = \sum_{i=0}^r\frac{\partial S}{\partial X_i}(X_i^2-X_i^1)$$
 and
 
-$$\frac{\partial}{\partial \alpha} \left[ (1-\alpha)S(X^1)+\alpha S(X^2) \right] = S(X^2)-S(X^1).$$
```

Comment:
I'm of the view that equations should have punctuation, the same as text. I.e. if an equation finishes a sentence, then it ends with a fullstops.